website/src/contribute/index.md

```diff
@@ -172,6 +174,9 @@ To contribute code, you need
    Use `R: @username` in the pull request to notify a reviewer.
 1. If you don't get any response in 3 business days, email the dev@ list to ask for someone to look at your pull
    request.
+1. Review feedback typically leads to follow-up changes. Please add these changes as additional "fixup" commits to the
+   existing PR/branch. This will allow reviewer(s) to track the incremental progress. After review is complete and the
+   PR accepted, multiple commits should be squashed (see [Git workflow tips](https://cwiki.apache.org/confluence/display/BEAM/Git+Tips)).
```

Comment:
The number of links into the Wiki is getting worrisome, as it is unstable by design. I wonder if there is another way to handle this?

Reply:
For this content, I'd rather have links that might go bad than not have them (or a vague check the wiki).